governance/program/src/state/proposal.rs

```diff
@@ -194,7 +194,11 @@ pub struct ProposalV2 {
     pub vote_threshold: Option<VoteThreshold>,
 
     /// Reserved space for future versions
-    pub reserved: [u8; 64],
```

Comment:
Thinking if it could be done line `[u32; 6] + [u8;8]`. I'm not sure what's best practice about this.

Reply:
the way you did it, is fine